### ISSUE TITLE
"eth_chainId" call returns 0x0 when actual node responds with 0x1

### ISSUE THREAD

**Issue by E1458** [emeraldpay/dshackle#68]
First, I had to add "eth_chainId" method to dshackle.yaml file. The documentation does not tell how to define multiple methods, so I tried many different ways, like just using multiple lines of "-name:" or something like this:      
```
      methods:
        enabled:
          - name: eth_chainId
      methods:
        enabled:
          - name: eth_getLogs
```
Dshackle runs fine without error message. But when I test with

`curl -X POST --insecure -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_chainId","params":[], "id":64}' http://localhost:8080/eth`

the response was

`{"jsonrpc":"2.0","id":64,"result":"0x0"}`

, which is different from actual node(Infura)'s response to the same query (0x1). This error is preventing me from using dshackle as an eth1 load balance for eth2 client. This never happened before the eth2.0 client's (Teku) mainnet version (which was published today). I assume the previous versions of Teku did not call "eth_chainId" method.

**Comment by E1458** [comment]
Teku engineer says "eth_chainId" was there long before the mainnet version.

**Comment by MysticRyuujin** [comment]
Hey @E1458 this has already been fixed, it's just awaiting a release

**Comment by splix** [comment]
In general the `eth_chainId` method is hardcoded and the answer is produced by Dshackle without touching an upstream. So you cannot make it enabled right now (but i've added it as a feature request #70). The fact that it produces the the wrong id is a bug, I also think it's fixed but I'll double check.

For multiple methods you just pass them like that:
```
      methods:
        enabled:
          - name: eth_chainId
          - name: eth_getLogs
```
I'm not sure the way how you described it is actually working, though maybe. 

It's good idea to add an example to the docs. If you can make an update, that would be great. And if you have noticed other things that are not explained well in the docs, I'd appreciate your suggestion.

**Comment by MysticRyuujin** [comment]
The wrong ID was fixed in #55

**Comment by splix** [comment]
dup #55
